README.md

```diff
@@ -65,188 +65,36 @@ See the [Tesuji Plasma design document](docs/tesuji_blockchain_design.md) for a
-Initialize the database with the following command.
-**CAUTION** This wipes the old data clean!:
-```bash
-rm -rf ~/.omg/data
-mix run --no-start -e 'OMG.DB.init()'
-```
-
-The database files are put at the default location `~/.omg/data`.
-You need to re-initialize the database, in case you want to start a new child chain from scratch!
-
-#### Start it up!
-* Start up geth if not already started.
-* Start Up the child chain server:
-
-```bash
-iex -S mix xomg.child_chain.start --config ~/config.exs
-```
-
-### Setting up a Watcher (a developer environment)
-
-This assumes that you've got a developer environment Child chain server set up and running on the default `localhost:9656`, see above.
-
-#### Configure the PostgreSQL server with:
-
-```bash
-sudo -u postgres createuser omisego_dev
-sudo -u postgres psql
-alter user omisego_dev with encrypted password 'omisego_dev';
-ALTER USER omisego_dev CREATEDB;
-```
-
-#### Configure the Watcher
-
-Copy the configuration file used by the Child chain server to `~/config_watcher.exs`
-
-```bash
-cp ~/config.exs ~/config_watcher.exs
-```
-
-You need to use a **different** location of the `OMG.DB` for the Watcher, so in `~/config_watcher.exs` append the following:
-
-```elixir
-config :omg_db,
-  leveldb_path: Path.join([System.get_env("HOME"), ".omg/data_watcher"])
-```
-
-#### Initialize the Watcher's databases
-
-**CAUTION** This wipes the old data clean!
-
-```bash
-rm -rf ~/.omg/data_watcher
-mix ecto.reset --no-start
-mix run --no-start -e 'OMG.DB.init()' --config ~/config_watcher.exs
-```
-
-#### Start the Watcher
-
-It is possible to run the watcher in two different modes:
-- The first is `security critical` mode.
-```bash
-iex -S mix xomg.watcher.start --config ~/config_watcher.exs
-```
-- The second is `security critical` + `convenience` mode.
-```bash
-iex -S mix xomg.watcher.start --convenience --config ~/config_watcher.exs
-```
+If the blockchain services are not already present on the host, docker-compose will attempt to build the image with the tag `elixir-omg:dockercompose` and continue to use that. If you want Docker to use the latest commit from `elixir-omg` you can trigger a fresh build by passing the `--build` flag to `docker-compose up --build`.
 
-See docs/TODO for more details about watcher modes.
+## Install on a Linux host & manual start up
+Follow the guide to **[install](docs/install.md)** the child chain server and watcher. Then use the guide in **[manual service startup](docs/manual_service_startup.md)** to stand up.
 
 ### Follow the demos
 After starting the child chain server and/or Watcher as above, you may follow the steps in the demo scripts.
 Note that some steps should be performed in the Elixir shell (iex) and some in the shell directly.
 
-To start a configured instance of the `iex` REPL, from the `elixir-omg` root directory do:
+To start a configured instance of the `iex` REPL, from the `elixir-omg` root directory inside the container do:
```

Comment:
I added that in because after the image is built and the container is running, there are two different `elixir-omg` repos (the repo gets copied into the container). It should be possible to run the `iex` commands outside of the container, but that assumes the developer has knowledge of how to install Elixir and connect to the service. By executing this inside of the container this process is simplified for someone who just wants to start testing and developing.